static/src/javascripts/bootstraps/media.js

```diff
@@ -214,8 +214,13 @@ define([
                 // preroll for videos only
                 if (mediaType === 'video') {
 
+                    if (showEndSlate && detect.isBreakpoint({ min: 'desktop' })) {
+                        initEndSlate(player, endSlateUri);
+                    }
+
                     player.fullscreener();
-                    // Init plugins
+
+                    // do ads last so if the ad blocker throws an exception it doesn't stop anything else
```

Comment:
I could, that's what I was wondering in the description, but I 've heard it's a big performance issue to do so.  I'm not sure I want to slow things down just to help out people with ad blockers, but it depends how bad the hit is.